sourcegraph/sourcegraph
# browser-ext: auto-create repos and refresh annotation data

+ if the user is viewing a supported file type (Go/Java), ensure the repo has been cloned
+ refereshVCS on interval and trigger re-render cycle when new (annotation) data is available
+ include visual build indicator on file header

Test plan:

- create a new github repository with a single go file (e.g. https://github.com/rothfels/dummygo)
- click on the .go file
- see that the build indicator says "Indexing"...
- ^^ wait for 1-2 minutes, see that annotation data eventually shows up on the page & the build indicator is updated
